components/tikv_util/src/codec/bytes.rs

```diff
@@ -278,11 +278,7 @@ pub fn decode_bytes_in_place(data: &mut Vec<u8>, desc: bool) -> Result<()> {
     }
 }
 
-/// Returns whether `encoded` bytes is encoded from `raw`.
-///
-/// # Panics
-///
-/// Panics if `encoded` is not valid
+/// Returns whether `encoded` bytes is encoded from `raw`. Returns `false` if `encoded` is invalid.
```

Comment:
If `encoded` is invalid, it must **not** be encoded from `raw`. Thus this scenario is already covered by the first sentence.

Reply:
It's just to clarify that this function can accept invalid encoded bytes :)